### ISSUE TITLE
Does region-level soft labels exist in Code ?

### ISSUE THREAD

**Issue by Yinyaommmm** [LINs-lab/RDED#8]
Hi, thank you for making the code public.
After reading the paper and code , however , I still have a question. I’d appreciate it if you could address my confusion.

In your paper <Chatper 4.3. Information Reconstruction of Patches - Labels reconstruction> , you mentioned that Inspired by this observation, we propose to **relabel the squeezed multi-patches** within the distilled images xj, thereby encapsulating the informative label for the distilled images.

But upon reviewing the code (validation/main.py) and (synthesize/main.py), it seems like there is no code for relabeling synthesized images. Did you really use region-level soft labels in your code? The most possible code(in validation/main.py) I think is : 
```python
            mix_images, _, _, _ = mix_aug(images, args)
            pred_label = student_model(images)
            soft_mix_label = teacher_model(mix_images)
            soft_mix_label = F.softmax(soft_mix_label / args.temperature, dim=1)
```
But it actually relabels the entire synthesized image rather than pathches.

Thanks for your time and outstanding work and look forward to your reply!
